### ISSUE TITLE
The font size increase is not permanent

### ISSUE THREAD

**Issue by r-valyar** [jegasus/chatlog-scaler#2]
I noticed that the font size in the chat log is not permanent. It affects existing messages, but it does not apply to new ones. is this default behavior?

**Comment by jegasus** [comment]
No, this isn't exactly a bug. It's just an issue I haven't found a good way around yet. 
The current strategy that works is to re-scale after sending new messages. When re-scaling, that forces all the messages published in the chatlog thus far to become the same size.
I'll soon figure out a cool way to solve this, but for now we're stuck with this dirty workaround.

**Comment by r-valyar** [comment]
A workaround is better than nothing :) Fingers crossed you find solution to that :)
